apps/prairielearn/src/models/variant.ts

```diff
@@ -102,11 +102,6 @@ export async function selectUserOwnsVariant({
   return await queryRow(sql.select_user_owns_variant, { user_id, variant_id }, z.boolean());
 }
 
-/**
- * TODO: We need to add checks for public question previews. That page should
- * only allow viewing of variants that are owned by the current user and that
- * were created on the public question preview page.
- */
```

Comment:
We already have checks in this function that check: 

- the user owns the variant
- if the page is for an instance question, the variant's instance question id must match

this ensures that no one can access variants _on the public page_  that they shouldn't be able to. So I think it makes sense to remove this comment.

The other problem related to public preview access is that instructors could access data from public question previews, but that is covered by the comment below. 

If we decide that we don't actually care about that case, we could remove that comment to.